cbor-file-magic.mkd

```diff
@@ -285,7 +285,20 @@ If the eight bytes are a problem on the wire (and they often are if CBOR is bein
 If the programs that read the contents of the file already expect to process all of the CBOR data items in the file (not just the first), then the CBOR Tag Sequence format may be easily retrofitted.
 
 The program involved may throw errors or warnings on the CBOR Tag Sequence if they have not yet been updated, but this may not be a problem.
-If it is, then consideration should be given to CBOR Tag Wrapped.
+
+There are situations where multiple objects may be concatenated into a single file.
+If each object is preceeded by a a CBOR Tag Sequence then there may be multiple Tag Sequences in the file.
+If it is, then consideration should be given to CBOR Tag Wrapped instead.
```

Comment:
On the other hand, keeping the interspersed headers may be exactly what the doctor ordered.